lib/cisco_node_utils/cmd_ref/bgp_af.yaml

```diff
@@ -59,8 +88,16 @@ dampen_igp_metric:
 
 dampening:
   auto_default: false
-  config_get_token_append: '/^dampening(?: (?:(\d+) (\d+) (\d+) (\d+)|route-map (.*)))?$/'
-  config_set_append: '<state> dampening <route_map> <decay> <reuse> <suppress> <suppress_max>'
+  cli_nexus:
+    config_get_token_append: '/^dampening(?: (?:(\d+) (\d+) (\d+) (\d+)|route-map (.*)))?$/'
+    config_set_append: '<state> dampening <route_map> <decay> <reuse> <suppress> <suppress_max>'
+  cli_ios_xr:
+    config_set: 'router bgp <asnum>'
+    config_set_append:
+      - ' address-family <afi> <safi>'
+      - ' <state> bgp dampening <decay> <reuse> <suppress> <suppress_max>'
+    config_get: 'show running-config router bgp | in bgp dampening'
```

Comment:
ruby test_bgp_af.rb -v  -- 192.168.122.245:57888 root lab       
/home/gvlaemyn/.rvm/gems/ruby-2.2.1/gems/grpc-0.11.0/lib/grpc/generic/client_stub.rb:173: warning: circular argument reference - parent
Run options: -v -- --seed 3405

# Running:


Node under test:
  - name  - 
  - type  - R-IOSXRV9000-RP
TestRouterBgpAF#test_collection_empty = 3.83 s = .
TestRouterBgpAF#test_redistribute = 78.16 s = .
TestRouterBgpAF#test_collection_not_empty = 4.15 s = .
TestRouterBgpAF#test_dampen_igp_metric = 2.50 s = S
TestRouterBgpAF#test_remove_route_policies = 15.07 s = .
TestRouterBgpAF#test_additional_paths = 8.16 s = .
TestRouterBgpAF#test_additional_paths_selection = 8.19 s = .
TestRouterBgpAF#test_network = 138.94 s = .
TestRouterBgpAF#test_advertise_l2vpn_evpn = 2.34 s = S
TestRouterBgpAF#test_next_hop_route_policy = 14.13 s = .
TestRouterBgpAF#test_client_to_client = 6.99 s = .
TestRouterBgpAF#test_next_hop_route_map = 2.62 s = S
TestRouterBgpAF#test_maximum_paths_ibgp = 20.27 s = .
TestRouterBgpAF#test_dampening = 12.65 s = .
TestRouterBgpAF#test_default_information_originate = 7.41 s = .
TestRouterBgpAF#test_feature_nv_overlay_evpn = 2.37 s = S
TestRouterBgpAF#test_maximum_paths = 21.81 s = .
TestRouterBgpAF#test_utils_delta_add_remove = 2.32 s = .

Finished in 351.919095s, 0.0511 runs/s, 0.3609 assertions/s.

  1) Skipped:
TestRouterBgpAF#test_dampen_igp_metric [test_bgp_af.rb:534]:
dampen-igp-metric unsupported on IOS XR


  2) Skipped:
TestRouterBgpAF#test_advertise_l2vpn_evpn [test_bgp_af.rb:523]:
L2VPN EVPN is unsupported on IOS XR


  3) Skipped:
TestRouterBgpAF#test_next_hop_route_map [test_bgp_af.rb:279]:
nexthop route-map drop_all unsupported on IOS XR


  4) Skipped:
TestRouterBgpAF#test_feature_nv_overlay_evpn [test_bgp_af.rb:839]:
nv overlay evpn unsupported on IOS XR

18 runs, 127 assertions, 0 failures, 0 errors, 4 skips
Coverage report generated for MiniTest to /home/gvlaemyn/cisco-network-node-utils/tests/coverage. 0.0 / 0.0 LOC (100.0%) covered.